### PR TITLE
Prefer infra-vlan configured on APIC

### PR DESCRIPTION
**Pull request by mandeepdhami**
Prefer infra-vlan configured on APIC if it conflicts the infra-vlan
configured in the input YAML file

### CODE REVIEW

File: provision/acc_provision/acc_provision.py [hunk 1]
```diff
@@ -202,19 +202,19 @@ def config_discover(config, prov_apic):
     if prov_apic is not None:
         apic = get_apic(config)
 
+    orig_infra_vlan = config["net_config"].get("infra_vlan")
```
Comment (gaurav-dalvi) [inline provision/acc_provision/acc_provision.py]: LGTM. I hope value setting to None wont create problem in the code below where it is referenced.

**Comment by gaurav-dalvi** [comment]
Do we need to change files here : https://github.com/noironetworks/acc-provision/tree/master/provision/testdata

File: provision/acc_provision/acc_provision.py (hunk 1 above)
Comment (mandeepdhami) [reply provision/acc_provision/acc_provision.py]: There is a use case where None is a valid value, so we need to handle that. And the following code should handle it.

**Comment by snaiksat** [comment]
We can enhance UTs as a follow up patch.

**Pull request merged**